### ISSUE TITLE
Some addons showing incorrect update status

### ISSUE THREAD

**Issue by Barleduq** [AcidWeb/CurseBreaker#350]
After my shenanigans this morning (which i talked on discord about), I still have the following problems that seem to be persistent.

Some addons are displaying 'update suppressed' when the addon is actually updated
- Below Average Items (from WoWI) shows as version 1.37. The .toc file on my computer shows 1.38
- BlizzMove (from github) shows 3.5.1, toc says 3.5.2
- TheUndermineJournal (wowi) shows 5.6.20221025, toc says 5.6.20221029
- RollHistory (github) shows v9.0.0.20731b, toc shows 10.0.0.221026

QuestGuru Sounds shows Modified. I just installed it today, and haven't done anything with it. 

Others are updating just fine. 

I had been seeing things with weird status, and thought it might relate to me having two terminal windows open, both with CB running. I exited both, and then re ran CB to give a final update, and then exited CB and closed the terminal window before playing for about 3.5 hours, 2.5 of that in raid, with frequent reloads. (not that I really expect that would have changed anything, but I'm giving you all my actions.)  After I left raid, I opened a terminal window and ran CB again, and this is the result.  These errors persisted, some went away. 

Ubuntu 22.04, CB v4.3.1, WoW prepatch as of 12:00 EST .

**Comment by AcidWeb** [comment]
`force_update` them all. This type of issues are expected if you restore older CurseBreaker.json.

**Comment by Barleduq** [comment]
DO you mean do a force_update that updates everythign at once?  I have force_updated everything at least 3-4 times.

**Comment by AcidWeb** [comment]
Make sure they are not pinned with `toggle pinning`

**Comment by Barleduq** [comment]
![CursebreakerToggling](https://user-images.githubusercontent.com/20568287/198891587-09f27b55-461f-4363-8ce6-85b3e365db59.png)
![CursebreakerForceUpdateREsults-AltoTUJ](https://user-images.githubusercontent.com/20568287/198891693-49b18bb2-9fdc-40e0-acf9-d1b723a83b01.png)

**Comment by Barleduq** [comment]
The others mentioned above are still showing Update Suppressed after an update command. After this update, Altoholic showed Update Suppressed, presumably because I had to remove files in order to update (by order of the addon author). Now, as you see, Altoholic and TheUndermineJournal are showing Modified, despite my not having touched them between the force update command and the status command.  

Strangely enough, doing a status after typing the above shows Altoholic and The Undermine Journal as up to date, but QuestGuru Sounds still shows as Modified, and the others still show Update Suppressed.  Something is strange - maybe a race condition of some sort?

**Comment by AcidWeb** [comment]
Something must edit addon directories between `force_update` and `status`. Do you have some kind of sync software? Anything that can modify wow addon directory contents?

**Comment by Barleduq** [comment]
No, I haven't had done any syncing since I stopped using 2 computers to play wow. I have addons I'm getting from e.g. Townlong-yak, but I installed those separately, late Saturday. Other than installing things by hyand that I have actually uninstalled from CurseBreaker - so CB shouldn't be aware of them outside of checking for orphans. Those commands in the image were literally one right after the other, as fast as I could type. 

I currently have exited battle.net and exited Lutris, which I use to manage WINE. I uninstalled QuestGuru Sounds, and have issued a force_update of everything.  And it now all shows updated.  What's different about doing a force_update of an individual package, and doing a force_update of everything?  

I'm clueless.  I'm closing this, since my original problem is apparently fixed.  Hopefully it all behaves again.
-Barleduq
